src/implementations/twig/components/checkbox/checkbox-item.html.twig

```diff
@@ -36,6 +37,7 @@
 {% set _helper_text = helper_text|default('') %}
 {% set _label = label|default('') %}
 {% set _css_class = 'ecl-checkbox' %}
+{% set _required_text = required_text|default('') %}
```

Comment:
Shouldn't it be '*' by default?
Also we could document this new parameter in the migration guide

Reply:
Changed a bit the logic, we never set a default for the required text so also in this case we define it as an empty string and we expect a `required_text` prop to be defined in the checkbox object when there is a single, required, checkbox.
An item has been added in the migration doc.